maps/tether/tether_virgo3b.dm

```diff
@@ -177,11 +177,28 @@ var/datum/planet/virgo3b/planet_virgo3b = null
 		WEATHER_HAIL = 5
 		)
 	observed_message = "It is snowing lightly."
+	effect_message = list(
+		"<I>The gentle breeze lifts tiny particles of falling snow past you.</I>",
+		"<I>Cool wind rushes over your senses as the breeze softly stirs and spirals around you.</I>",
+		"<I>A momentary pause in wind leaves the air still before it finds it's peaceful rythm again.</I>",
+		"<I>Blanketed cold envelops you as the wind carries it's chilled embrace.</I>"
```

Comment:
rhythm, not rythm
its, not it's (possessive vs contraction of it is/it has)

Reply:
fixed <3